### DISCUSSION TITLE
Looking to try out v4.0 (on macOS) but I'd like to keep 3.0 installed in case I want to stick with the old version. Will installing/running 4.0 cause anything to break with my 3.0 install?

### DISCUSSION THREAD

**Discussion by dmattera** [transmission/transmission#5012] [Q&A]
### What is the issue?

I made backups of the following files/folders that I know 3.0 uses for preferences/data/etc so I can restore them if needed but just wanted to see if there's anything else I should do in the event I want to revert back after trying 4.0:

- ~/Library/Preferences/org.m0k.transmission.plist
- ~/Library/Application Support/Transmission

Thanks!

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0

**Comment by dantearmok** [comment]
I've got 3.0, 4.0.1, and a nightly, all installed in /Applications (just name them differently).  No need to fool with the preferences or library list.  Just be sure to run only one at a time.

hth,
- Dan.
